packages/shared/lib/core/profile/actions/profiles/checkAndMigrateProfiles.ts

```diff
@@ -156,6 +156,9 @@ function persistedProfileMigrationToV9(existingProfile: IPersistedProfile): void
 }
 
 function persistedProfileMigrationToV10(existingProfile: IPersistedProfile): void {
+    const network = NETWORK?.[existingProfile?.networkProtocol]?.[existingProfile?.networkType]
+    existingProfile.network = network
+
```

Comment:
See comment on #6494 about duplicating this logic / checking for `network` property